src/elements/content-sidebar/Sidebar.js

```diff
@@ -102,13 +113,35 @@ class Sidebar extends React.Component<Props, State> {
             this.setForcedByLocation();
             this.setState({ isDirty: true });
         }
+
+        if (activeAnnotationId !== prevActiveAnnotationId) {
+            this.handleAnnotationSelect(activeAnnotationId);
+        }
     }
 
     getUrlPrefix = (pathname: string) => {
         const basePath = pathname.substring(1).split('/')[0];
         return basePath;
     };
 
+    /**
+     * Handle annotation selection
+     *
+     * @param {string} annotationId - The id of the annotation being selected
+     * @return {void}
+     */
+
+    handleAnnotationSelect = (annotationId: string | null): void => {
+        if (!annotationId) {
+            return;
+        }
+
+        const { history } = this.props;
+
+        const urlPrefix = this.getUrlPrefix(history.location.pathname);
+        history.replace(`/${urlPrefix}/annotations/${annotationId}`);
```

Comment:
Do we want to `push` instead of `replace` here?

Reply:
Is this URL going to suffice for our purposes? Will we be able to determine the version we need to load?

Reply:
No, but I talked with the BE team and they have not worked through the versions url yet and what it will look like.